### PR TITLE
Ixs 2864 enable the submit button when no error

### PR DESCRIPTION
**Pull request by harryng22**

### CODE REVIEW

**Review by witchaotic — APPROVED** [review]

File: src/components/LaunchpadIssuance/utils/form.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
-export const isSubmitDisabled = (errors: any, touched: any) => {
+export const isSubmitDisabled = (errors: any) => {
   const hasErrors = Object.keys(errors).length > 0
```
Comment (witchaotic) [inline src/components/LaunchpadIssuance/utils/form.ts:2]: Please recheck the task, I left some comments there. you need to leave the `notTouched` check. 
you should disable it only if there are visible errors, basically the `isDraftDisabled` check. if there are no visible errors, the user should be able to click, so the whole form validation would be triggered(all touched). 
this is really easy to reproduce with new forms - some fields are empty, the user cannot click on the submit button, yet it doesn't know what it requires for submitting. 
we still show only the touched errors, so they can work on the form and save draft as many times as they want. but when clicking submit they should either see all errors or submit the form(if no errors).
I hope I explained it well, please check the task comment as well.

**Review by witchaotic — CHANGES_REQUESTED** [review]

File: src/components/LaunchpadIssuance/utils/form.ts (hunk 1 above)
Comment (harryng22) [reply src/components/LaunchpadIssuance/utils/form.ts]: I think it's not intuitive for users to know they need to touch (change something) to enable the submit button after they save the form, and come back later. So I delete the touched property.
I changed code back to before.

Comment (witchaotic) [reply src/components/LaunchpadIssuance/utils/form.ts]: yes, it's not intuitive, that's why we should **enable** the submit button if there are no **visible** errors, so the user can click on it and see all the errors.   
you can also add a message with `showError` for the user to check his form again in that case.

Comment (witchaotic) [reply src/components/LaunchpadIssuance/utils/form.ts]: @harryng22 this one

File: src/components/LaunchpadIssuance/utils/form.ts [hunk 2]
```diff
@@ -13,7 +11,8 @@ const getIsDisabled = (message?: string) => {
   }
   return false
 }
-export const isDraftDisabled = (errors: any, touched: any) => {
```
Comment (witchaotic) [inline src/components/LaunchpadIssuance/utils/form.ts:16]: for `isSubmitDisabled` you don't need the required exception, all errors count. basically the getIsDisabled should not be called or always return true

**Review by witchaotic — CHANGES_REQUESTED** [review]

**Review by witchaotic — APPROVED** [review]

**Pull request merged**